maps/ministation/jobs/command.dm

```diff
@@ -0,0 +1,159 @@
+/datum/job/ministation/captain
+	title = "Captain"
+	supervisors = "your profit margin, your conscience, and the Trademaster"
+	outfit_type = /decl/hierarchy/outfit/job/ministation/captain
+	min_skill = list(
+		SKILL_LITERACY = SKILL_ADEPT,
+		SKILL_WEAPONS  = SKILL_ADEPT,
+		SKILL_SCIENCE  = SKILL_ADEPT,
+		SKILL_PILOT    = SKILL_ADEPT
+	)
+	max_skill = list(
+		SKILL_PILOT   = SKILL_MAX,
+		SKILL_WEAPONS = SKILL_MAX
+	)
+	skill_points = 30
+	head_position = 1
+	department_refs = list(DEPT_COMMAND)
+	total_positions = 1
+	spawn_positions = 1
+	selection_color = "#1d1d4f"
+	req_admin_notify = 1
+	access = list()
+	minimal_access = list()
+	minimal_player_age = 14
+	economic_power = 20
+	ideal_character_age = 70
+	guestbanned = 1
+	must_fill = 1
+	not_random_selectable = 1
+
+/datum/job/ministation/captain/equip(var/mob/living/carbon/human/H)
+	. = ..()
+	if(H.client)
+		H.client.verbs += /client/proc/freetradeunion_rename_company
```

Comment:
pretty sure what this should be a mob verb, not client (if we giving verb to client they can use it after ghosting)

Reply:
Thank you for bringing this to my attention, that is a very valid concern.  If this requires changing, it appears to require changing on Tradeship as well.  I'm still learning the very fundamentals of this, so I will attempt to look at this in a future PR, with apologies.